packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -562,3 +562,14 @@ export type SubscriptionActions = {
     terminate: SubscriptionAction[];
     system: SubscriptionAction[];
 };
+
+export type CacheTag = { type: CacheTagTypes; id?: string };
+
+export enum CacheTagTypes {
```

Comment:
Enums shouldn't be plural: `CacheTagTypes` -> `CacheTagType`

Reply:
Fixed